release notes/v0.55.2.md

```diff
@@ -0,0 +1,8 @@
+k6 `v0.55.2` is here 🎉! This release includes:
+
+- Dependency updates for `golang.org/x/net`.
+- Fixing a packaging problems in `v0.55.1`
+
+## Maintenance and internal improvements
+
+- [#4134](https://github.com/grafana/k6/pull/4134) Updates `golang.org/x/net` which contains a fix for CVE-2024-45338.
```

Comment:
How about something like that? Technically, it's not about fixing the CVE

```suggestion
k6 `v0.55.2` is a patch release that fixes packaging issue:

- Fixing a packaging problems in `v0.55.1`

## Maintenance and internal improvements

- [#4143](https://github.com/grafana/k6/pull/4143) Fix k6packager workflow.
```

There are no functional changes in k6 compared to v0.55.1.

Reply:
note: updated to a shorter version